src/vtl.ts

```diff
@@ -686,7 +691,7 @@ export abstract class VTL {
     } else if (isVariableStmt(node)) {
       // variable statements with multiple declarations are turned into multiple variable statements.
       const decl = node.declList.decls[0];
-      return this.evalDecl(decl);
+      return decl ? this.evalDecl(decl) : undefined;
```

Comment:
What is the result of returning `undefined`?

Reply:
The contract says that `Stmt`s return void. A variable stmt shouldn't return a value.